x-pack/plugins/cases/public/components/custom_fields/index.tsx

```diff
@@ -39,7 +39,7 @@ const CustomFieldsComponent: React.FC<Props> = ({
   customFields,
 }) => {
   const { permissions } = useCasesContext();
-  const canAddCustomFields = permissions.create && permissions.update;
```

Comment:
Could you please update `x-pack/plugins/cases/public/components/custom_fields/index.test.tsx` to test this?

I think you can pass the permissions to `createAppMockRenderer()`.

(And also `x-pack/plugins/cases/public/components/templates/index.test.tsx`)

Reply:
No need to add tests here anymore.

Reply:
yeah, i wrote this comment before reaching the other conclusion. I'll mark it as resolved.